content/de/contact.md

```diff
@@ -22,6 +23,7 @@ Der Datenanfragen.de e.&nbsp;V. ist im Vereinsregister des Amtsgerichts Braunsch
 
 **Verantwortlich für den Inhalt nach § 18 Abs. 2 MStV:**  
 Benjamin Altpeter, Schreinerweg 6, 38126 Braunschweig, Deutschland
+</address>
```

Comment:
I'd say the last two paragraphs aren't really contact info anymore, but I guess one could argue about that.